### PR TITLE
{CI} Fix code search pattern in version bot

### PR DESCRIPTION
**Pull request by AllyW**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: scripts/ci/release_version_cal.py [hunk 1]
```diff
@@ -37,11 +37,11 @@
 def extract_module_history_update_info(mod_update_info, mod):
     """
     re pattern:
-    --- a/src/monitor-control-service/HISTORY.rst
-    +++ b/src/monitor-control-service/HISTORY.rst
+    --- a/src/monitor-control-service/HISTORY.rst/md
+    +++ b/src/monitor-control-service/HISTORY.rst/md
```
Comment (wangzelin007) [inline scripts/ci/release_version_cal.py:41]: It looks like you want to match HISTORY.rst or HISTORY.md?
```suggestion
    --- a/src/monitor-control-service/HISTORY.(rst|md)
    +++ b/src/monitor-control-service/HISTORY.(rst|md)
```

File: scripts/ci/release_version_cal.py [hunk 2]
```diff
@@ -37,11 +37,11 @@
...
     """
     mod_update_info["history_updated"] = False
-    module_history_update_pattern = re.compile(r"\+\+\+.*?src/%s/HISTORY.rst" % mod)
+    module_history_update_pattern = re.compile(r"\+\+\+.*?src/%s/HISTORY.[rstmd]" % mod)
```
Comment (wangzelin007) [inline scripts/ci/release_version_cal.py:44]:
```suggestion
    module_history_update_pattern = re.compile(r"\+\+\+.*?src/%s/HISTORY\.(rst|md)" % mod)
```

File: scripts/ci/release_version_cal.py (hunk 1 above)
Comment (AllyW) [reply scripts/ci/release_version_cal.py]: yes, found HISTORY.md in some extension module

**Review by wangzelin007 — APPROVED** [review]

**Pull request merged**